PANDORA/cmd_pandora.py

```diff
@@ -226,4 +226,4 @@ def cmd_run_wrapper():
         IDs_col=args.targets_id_column, peptides_col=args.peptides_column,
         allele_name_col=args.allele_name_column, anchors_col=args.anchors_column,
         n_loop_models=args.loop_models, collective_output_dir=args.output_path,
-        clip_C_domain=args.clip_C_domain)
\ No newline at end of file
+        clip_C_domain=args.clip_C_domain, use_netmhcpan=True)
```

Comment:
Could I please ask you to set the use_netmhcpan argument to False and add an argument to the parser to turn it to true? Although I agree that in most of the cases a user wants to use netmhcpan, it is only an optional dependency and it would cause this script to crash in case a user does not have it installed.
If it's an issue for you I can also commit directly to your branch and take care of it.

Reply:
I think it's best if you add that yourself as I unfortunately don't really have a dev environment set up and was rather just adding quick fixes.
The reason I did it like this is that it seems to me that one either provided anchors (-k) via the .tsv, in which case netMHCpan is not tried or without -k there is no alternative to using netMHCpan.
I probably missed some edge case?

Reply:
Sure, I can easily do it.

A user might also want (for MHC-I only) to simply model the cases by using canonical anchoring, although I admit it's not the best, as it would always be better to predict the anchors somehow. 
I will make sure I also add some warnings about it.